### PR TITLE
Check requirements upon deploy

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

File: web/.htaccess [hunk 1]
```diff
@@ -158,12 +158,12 @@ AddEncoding gzip svgz
     # Serve gzip compressed CSS files if they exist and the client accepts gzip.
     RewriteCond %{HTTP:Accept-encoding} gzip
     RewriteCond %{REQUEST_FILENAME}\.gz -s
-    RewriteRule ^(.*css_[a-zA-Z0-9-_])\.css$ $1\.css\.gz [QSA]
+    RewriteRule ^(.*css_[a-zA-Z0-9-_]+)\.css$ $1\.css\.gz [QSA]
```
Comment (amitaibu) [inline web/.htaccess:161]: Related to Drupal core upgrade?

**Review by amitaibu — APPROVED** [review]
Thanks. Screenshot and MWG

File: web/.htaccess (hunk 1 above)
Comment (AronNovak) [reply web/.htaccess:161]: It is.

**Comment by AronNovak** [comment]
image

**Comment by amitaibu** [comment]
What is the Unsupported release? Meaning, modules are not up to date? If so, I'd relax this, as sometimes we'd do deploys before upgrading modules

**Comment by AronNovak** [comment]
@amitaibu Nope. Unsupported release is not oudated release. It means that the maintainer marked explicitly as Unsupported. Only 2 packages were like that. Simple outdated package does not throw a requirement error.

**Pull request merged**